src/components/StepProgress.vue

```diff
@@ -105,14 +106,16 @@ ul.progressbar {
 		background: var(--color-gray);
 	}
 	&::before {
-		color: var(--color-white);
-		background: var(--color-primary);
+		color: var(--color-black);
+		background: var(--color-white);
+		border-style: solid;
 	}
 }
 .progressbar li.done {
 	&::before {
 		color: var(--color-white);
-		background: var(--color-success);
+		// background: var(--color-success);
```

Comment:
```suggestion
		background: var(--color-success);
```
Would it bother people with colour blindness if active elements were nevertheless coloured? Then the experience for people with colour vision would hardly be changed.

Reply:
👍 I would suggest the same here, is this possible to keep the experience for people with color vision?

Reply:
Yes, I agree @domon